packages/mdc-drawer/README.md

```diff
@@ -100,22 +153,22 @@ Dismissible drawers are by default hidden off screen, and can slide into view. D
     </div>
   </header>
 
-  <div class="mdc-drawer-app-content">
+  <div class="mdc-drawer__app-content">
```

Comment:
I had this as `-app-content`, since I thought `__app-content` meant that it is contained within `mdc-drawer`. Does BEM say otherwise?

Reply:
Makes sense. Changed it back to `-app-content`.